io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,80 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "librewolf",
+    "rename-appdata-file": "io.gitlab.librewolf-community.appdata.xml",
+    "rename-icon": "io.gitlab.librewolf-community",
+    "modules": [
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        {
+            "name": "librewolf",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/lib",
+                "mv librewolf.tar.bz2 /app/lib/librewolf",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/browser/chrome/icons/default/default128.png /app/usr/share/icons/hicolor/128x128/apps/io.gitlab.librewolf-community.png",
+                "install -D -m644 /app/lib/librewolf/io.gitlab.librewolf-community.appdata.xml -t /app/share/metainfo",
+                "install -D -m644 io.gitlab.librewolf-community.desktop -t /app/share/applications",
+                "install -D -m755 librewolf.sh /app/bin/librewolf"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/librewolf-community/browser/linux/-/jobs/1267046474/artifacts/raw/LibreWolf-88.0.1-2.aarch64.tar.bz2",
+                    "sha256": "660284b741f6d2507e5d1eb3d423ad96577f59b7902c3028d189742c8cdb7424",
+                    "dest": "librewolf.tar.bz2",
+                    "strip-components": 0,
+                    "only-arches": [
+                        "aarch64"
+                    ]
+                },
```

Comment:
Consider using flatpak-external-data-checker or some GitHub Action to update it automatically.

Reply:
That sounds like a good idea. I'd take care of this somewhere down the line, if that was OK with you – or should this be implemented immediately?

Reply:
It can wait, I mention it for your own convenience.